openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -617,6 +618,7 @@ port_INLINE bool ieee154e_processIEs(OpenQueueEntry_t* pkt, uint16_t* lenIE) {
    uint16_t              len;
    uint16_t              sublen;
    PORT_SIGNED_INT_WIDTH timeCorrection;
+   bool                  synchronizeSlotoffset;

```

Comment:
synchronizeSlotoffset has been changed to f_asn2slotoffset and some comments have been added to document its usage.